### ISSUE TITLE
Feature request: Color picker

### ISSUE THREAD

**Issue by stdword** [yoyurec/logseq-awesome-links#13]
**Idea**
Despite of very cool [service](https://enes.in/sorted-colors) to get color names (link from `Readme`), it will be very handy to pick colors inside Logseq (and without Internet connection).

**Inspiration screenshots:**
1. `References styles` plugin:

<img width="400" src="https://user-images.githubusercontent.com/1984175/196701266-c5196966-e052-4640-b6c7-cca937a1cd75.png">

2. `Developer tools` from browsers:
<img src="https://user-images.githubusercontent.com/1984175/196700342-b3203c77-8f55-4c51-b20b-abb3b9d7787f.png" width="40%">

**Comment by yoyurec** [comment]
* References styles has own settings window, used by lot of devs before were introduced Logseq native settings
* there is no possibily to (easy) modify input on props editing cose it's just simple HTML textarea
* i thougt about "colorpicker will be nice to have", but it's in bottom of my featurelist... and i have no yet any idea how to implement this. PRs are welcome ))

**Comment by stdword** [comment]
As an implementation idea:
If it is hard to implement color picker field inside the text area — may be there is another place?
As an brainstorming: near the page title; right-aligned from page title; under the page props bullet point; no ui, but command to choose color...
Color picker field can be visible at 5% alpha channel without hover to not interract with theme and modify page color property when a color value chosen...

PS sure, it is low level as it is «setup only once functionality»

**Comment by yoyurec** [comment]
a lof of pain in implementation and tons of aside issues and false-positive trigerrings for such small feature
